packages/p2p/src/components/modal-manager/modal-manager-context-provider.tsx

```diff
@@ -39,26 +39,26 @@ const ModalManagerContextProvider = (props: React.PropsWithChildren<{ mock?: TMo
      * @param {key} string - the key to specify when persisting the local state, by default you should specify the local state name
      * @param {default_state} - the value you want the state to be initially
      */
-    const useSavedState = (key, default_state) => {
+    const useSavedState = (key: string, default_state: string[]) => {
```

Comment:
`non-blocking:` i changed a bit here as well and fixed the typing issue for it but i dont think it'll cause any huge conflicts when we merge quick add modal as well 🙏